README.md

```diff
@@ -11,50 +11,84 @@
 
 ## Setting up for local development
 
-### Prerequisites
+> [!IMPORTANT]  
+> This README assumes you have Docker installed. If you do not, please read [this](https://docs.docker.com/engine/install/) manual, based on your operating system. It will cover setting up and testing your very own Docker instance (lucky you!)
 
-This README assumes you have Docker installed. If you do not, please read [this](https://docs.docker.com/engine/install/) manual, based on your operating system. It will cover setting up and testing your very own Docker instance (lucky you!)
+### Requirements
+Node: `7.0.1`
+<br/>
+Docker: `27.3.1`
 
 ### Steps
 
 #### Cloning and initial setup
 
-1. Clone the repository by executing `git clone https://github.com/clowder-cloud/backend.git`
-2. Enter the cloned repository locally, `cd backend`
-3. Install all dependencies, `npm i`
+1. Clone the repository with
+```bash
+git clone https://github.com/clowder-cloud/backend.git
+```
+2. Enter the cloned repository locally, typically with
+```bash
+cd backend
+```
+3. Install all dependencies with
+```bash
+npm i
+```
 
 #### Setting up the database
 
-1. Ensure that Docker is started. This is down to how you have installed Docker. You can test Docker is online by running `docker info`
-2. Run `npm run setup-db`.
-3. Run `docker ps` to see the containers, and copy the container ID
-4. Run `npm run get-docker-ip [CONTAINER_ID]`, replacing `[CONTAINER_ID]` with the one copied from the previous step
-5. Save the IP output from the above command - It will be used later
-
-#### Setting up the environment for local development
+1. Ensure your Docker instance is started by running
+```bash
+docker info
+```
+2. Compose the Docker container with
+```bash
+npm run setup-db
+```
+3. Copy the containers IP, which is retrieved by running
+```bash
+npm run get-docker-ip
+```
+
+#### Creating the environment
 
 1. Create a new `.env` file at the root of the project
-2. Refer to the `example.env` file, replacing values where required
-   1. The `DATABASE_URL` will be constructed of values from `docker-compose.local.yaml` and the output of previous steps.
-      1. If you have not changed any values, you can copy & paste this into your `.env` - `DATABASE_URL="postgresql://root:root@HOST:5432/clowder"` and skip to step 5
-      2. `POSTGRES_USER` will be our `USERNAME`
-      3. `POSTGRES_PASSWORD` will be our `PASSWORD`
-      4. `POSTGRES_DB` will be our `DATABASE`
-      5. The IP copied from [Setting up the database](###-Setting-up-the-database) will be our `HOST`
-
-#### Migration and Seeding
+2. Refer to the `example.env` for what KEY:VALUE pairs you need
+    1. For `DATABASE_URL`, the `DATABASE`, `USERNAME` and `PASSWORD` can be found in `docker-compose.local.yaml`. The `HOST` will be the result of `npm run get-docker-ip`
```

Comment:
There's only one item so this should be a paragraph not a list item.

Reply:
This is for future proofing - in case we plan to add anything else to the .env